1.Data-exploration/L1000_cell_painting_comparison/.ipynb_checkpoints/L1000_CP_comparison_visualization-checkpoint.ipynb

```diff
@@ -0,0 +1,221 @@
+{
```

Comment:
`.ipynb_checkpoints` files should not be included. Please remove this file and add `.ipynb_checkpoints/` to your `.gitignore`.